### PR TITLE
Fix grammar and formatting in Window docs

### PR DESCRIPTION
**Pull request by gavinmorrow**

### CODE REVIEW

File: docs/reST/ref/window.rst [hunk 1]
```diff
@@ -61,8 +61,8 @@
 
    Event behavior if multiple Windows are created: When the close button is
    pressed, a ``WINDOWCLOSE`` event is sent. You need to explicitly destroy
-   the window. Note that the event ``QUIT`` will only be sent if all Window
-   has been destroyed.
+   the window. Note that the event ``QUIT`` will only be sent if all Windows
```
Comment (bilhox) [inline docs/reST/ref/window.rst:64]: if it's plurial, then it should not be capitalized as well.

**Review by bilhox — CHANGES_REQUESTED** [review]

File: docs/reST/ref/window.rst (hunk 1 above)
Comment (oddbookworm) [reply docs/reST/ref/window.rst:64]: I would actually suggest doing something like this instead
```rst
\`\`Window\`\`s
```

Comment (gavinmorrow) [reply docs/reST/ref/window.rst:64]: I was trying to follow the example of the rest of the file, where Window is capitalized (including plurals). In the [`Surface`](https://pyga.me/docs/ref/surface.html#:~:text=Normal%20Surfaces%20should%20not%20require%20the%20masks%20argument) docs, the same convention is followed.

Comment (gavinmorrow) [reply docs/reST/ref/window.rst:64]: @bilhox @oddbookworm what do you think?

Comment (bilhox) [reply docs/reST/ref/window.rst:64]: @gavinmorrow I would follow what oddbookworm suggested.

**Review by ankith26 — APPROVED** [review]
LGTM, thanks for the PR :tada:

**Review by zoldalma999 — APPROVED** [review]
LGTM, Thanks!

**Comment by gavinmorrow** [comment]
It looks completely wrong, does anyone know why? I'm not familiar with `.rst`.

Screenshot 2024-11-02 at 19 16 25

**Comment by gavinmorrow** [comment]
oh yeah didn't see #3205

**Pull request merged**